resources/DefaultStyle.json

```diff
@@ -14,6 +14,7 @@
     "ShadowColor": [20, 20, 20],
     "FontColor" : "white",
     "FontColorFaded" : "gray",
+    "FontFamily": "Microsoft YaHei",
```

Comment:
The default font should be the default font of Qt: "Segoe UI"

(This font ships with Qt, so we know it's safe to use)